easybuild/easyblocks/n/nwchem.py

```diff
@@ -199,6 +208,9 @@ def configure_step(self):
             libmpi = "-lmpichf90 -lmpich -lopa -lmpl -lrt -lpthread"
         else:
             raise EasyBuildError("Don't know how to set LIBMPI for %s", mpi_family)
+        if self.cfg['armci_network'] in ["OPENIB"]:
+            libmpi += " -libumad -libverbs -lpthread"
+        #if LooseVersion(self.version) <= LooseVersion("6.5"):
```

Comment:
That #if Looseversion (both of them) should probably be activated but i haven't had time to verify that it works if i do...